cpu/kinetis/kinetis-info.mk

```diff
@@ -31,16 +31,16 @@ export KINETIS_SPEED := $(word 11, $(KINETIS_INFO))
 
 ifeq ($(KINETIS_CORE), Z)
   # Cortex-M0+
-  export CPU_ARCH = cortex-m0plus
+  CPU_ARCH = cortex-m0plus
```

Comment:
while you are at it, you could also unexport the CPU_FAMILY variable below (and maybe rename it to CPU_FAM for consistency ?).
It's only used in included makefiles:
```
git grep CPU_FAMILY
boards/common/frdm/Makefile.include:export OPENOCD_CONFIG ?= $(RIOTBOARD)/common/frdm/dist/old-openocd-$(CPU_FAMILY).cfg
boards/common/frdm/Makefile.include:export OPENOCD_CONFIG ?= $(RIOTBOARD)/common/frdm/dist/openocd-$(CPU_FAMILY).cfg
cpu/kinetis/kinetis-info.mk:  export CPU_FAMILY = klx
cpu/kinetis/kinetis-info.mk:  export CPU_FAMILY = kx
cpu/kinetis/kinetis-info.mk:  export CPU_FAMILY = kx
```

Reply:
> while you are at it, you could also unexport the CPU_FAMILY variable below

`CPU_FAMILY` is not the same as `CPU_FAM`, for this I would actually wan't to completely re-implement the parsing for `Kinetis`. Here I would rather limit the changes to `CPU_FAM` and `CPU_FAMILY`.

> (and maybe rename it to CPU_FAM for consistency ?).

`KINETIS_SERIES` is more of an analog to `CPU_FAM`, but as I said I wan't to deal with all those `KINETIS` specific variables in one separate go.


Most of `KINETIS` variables do not need to be exported (actually I think all of them).

Reply:
> Most of KINETIS variables do not need to be exported (actually I think all of them).

Yes probably. And CPU_FAMILY, which is only defined in this file, is one of them. If unexport it in this PR, then the static test becomes simpler: `CPU_FAM` instead of `CPU_FAM CPU_FAM$`.

Reply:
Ok I'll do it if you insist.